server/src/main/java/model/shooting/field/DamageFieldState.java

```diff
@@ -0,0 +1,30 @@
+package model.shooting.field;
```

Comment:
That's not model ! That's model/service, so separete this

Reply:
after the separation the reuse of the BoardField would become easier

Reply:
For future consideration.